### PR TITLE
Java play framework

### PR DESCRIPTION
**Pull request by Zohar2506**

### CODE REVIEW

File: java-play-framework/2.5.x/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    $ ./play-java-starter-example -Dplay.file=../conf/application.conf
+    ```
+
+## Rookout Integration explained
+This example make sure to set up Rookout`s SDK as Java Agent for Play Framework
+  1. Add Java Agent plugin
+    ```
+    addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4")
+    ```
+  1. Enable Java Agent
```
Comment (itielshwartz) [inline java-play-framework/2.5.x/README.md]: 1 1 1

File: java-play-framework/2.5.x/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    ```
+    lazy val root = (project in file(".")).enablePlugins(PlayJava, JavaAgent)
+    ```
+  1. Set rook as the Java Agent
+    ```
+    javaAgents += "com.rookout" % "rook" % "0.1.9" % "dist"
+    ```
+
+## Running with sbt run
+Create lib folder and download Rookout`s rook module
```
Comment (itielshwartz) [inline java-play-framework/2.5.x/README.md]: Not sure i understand this

File: java-play-framework/2.5.x/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Run the Rookout agent:
+    ``` bash
+    $ docker run -p 7486:7486 -e "ROOKOUT_TOKEN=<Your-Token>" rookout/agent
+    ```
+## Running with sbt dist
+
+1.  Run the following commands:
+    ``` bash
+    $ cd 2.5.x
```
Comment (itielshwartz) [inline java-play-framework/2.5.x/README.md]: Don't you like a makefile: make build

File: java-play-framework/2.5.x/README.md (hunk 1 above)
Comment (Zohar2506) [reply java-play-framework/2.5.x/README.md]: It automatically number them

File: java-play-framework/2.5.x/README.md (hunk 2 above)
Comment (Zohar2506) [reply java-play-framework/2.5.x/README.md]: You can run with sbt run or package it with sbt dist... the 2 options are familiar for those who use sbt

File: java-play-framework/2.5.x/README.md (hunk 3 above)
Comment (Zohar2506) [reply java-play-framework/2.5.x/README.md]: Ill make one

**Pull request merged**